Sources/Components/Expandable/Expandable.swift

```diff
@@ -0,0 +1,142 @@
+import SwiftUI
+
+extension Warp {
+    /**
+
+    Expandable allows users to interact with expandable/collapsible sections of content. Also known as “accordion”
+
+    **When to use**
+
+     Use the Expandable component when presenting content that can be optionally revealed to users.
+     Consider it for long content that may overwhelm users if displayed all at once.
+
+     **Usage**
+
+     Provide the `Warp.Expandable` with:
+
+     - A title.
+     - A subtitle.
+     - A `Warp.ExpandableStyle`, choose from:
+        - `default`: for a non bordered expandable.
+        - `box`: for a bordered expandable with rounded corners.
+        - `boxBleed`: for a bordered expandable with sharp corners.
+     - A boolean `isAnimated` to determine if the expand/collapse transition is animated or not. **Optional** _default value is `true` if none is specified_.
+     - A `ColorProvider`. **Optional:** _default is read from `Config.colorProvider` if none is specified_.
+     */
+    public struct Expandable: View {
+        /**
+        **Collapsed by default**
+
+         Expendables begins in the collapsed state with all content panels closed.
+         Starting in a collapsed state gives the user a high-level overview of the available information.
+        */
+        @State private var isExpanded: Bool = false
+
+        let title: String
+        let subtitle: String
+        let style: Warp.ExpandableStyle
+        let isAnimated: Bool
+
+        let colorProvider: ColorProvider
+
+        public init(
+            title: String,
+            subtitle: String,
+            style: Warp.ExpandableStyle,
+            isAnimated: Bool = true,
+            colorProvider: ColorProvider = Config.colorProvider
+        ) {
+            self.title = title
+            self.subtitle = subtitle
+            self.style = style
+            self.isAnimated = isAnimated
+            self.colorProvider = colorProvider
+        }
+
+        public var body: some View {
+            VStack(spacing: Warp.Spacing.spacing200) {
+                alwaysVisibleView
+
+                if isExpanded {
+                    expandableView
+                }
+            }
+            .frame(maxWidth: .infinity)
+            .padding(Warp.Spacing.spacing200)
+            .background(style.backgroundColor(using: colorProvider))
+            .cornerRadius(style.cornerRadius)
+        }
+
+        @ViewBuilder private var alwaysVisibleView: some View {
+            HStack {
+                Warp.Text(
+                    title,
+                    style: .bodyStrong,
+                    color: colorProvider.expandableTitleText
+                )
+
+                if style != .default {
+                    Spacer()
+                }
+
+                Image("icon-expandable-chevron", bundle: .module)
```

Comment:
It's possible to use the SFSymbol 'chevron.down' for this.

Reply:
But do we want that? Don't we want to use our own?

Reply:
We have had some discussions around this and what's the best practice. If we use SFSymbol then we get some accessibility features for free and for example the icon will be dynamic and get bigger sizes. But in some places we may not want that.
Since we have not started defining icons we don't have a preference here. Maybe we will later on.

For now I would say if it makes sense for the icon to stay the same size use custom icon, if it make sense for it to be dynamic use SFSymbol.